.cargo/config_fast_builds.toml

```diff
@@ -134,6 +134,7 @@ rustflags = [
 # rustup component add llvm-tools
 # ```
 linker = "rust-lld.exe"
+rustdocflags = ["-Crust-lld.exe"]
```

Comment:
I'm confused, doesn't `-C` just mean a generic codegen option and not the linker specifically? Shouldn't this be something like `-Clink-args=-fuse-ld=rust-lld.exe`?

Reply:
Oops I think I messed up when copying from my local `config.toml` (which is probably why I didn't notice it being wrong). It was supposed to be `-Clinker=rust-lld.exe`, but I'm not sure what the difference with `-Clink-args=-fuse-ld=rust-lld.exe` would be.

Reply:
`-Clink-args=-fuse-ld=rust-lld.exe` is likely a Clang / GCC-only command, so I would just focus on finding the Windows MSVS version.

Reply:
I've fixed the typo to `-Clinker=rust-lld.exe`. `-Clink-args=-fuse-ld=rust-lld.exe` doesn't appear to work, maybe `-fuse-ld` is an argument specific to GCC and Clang's linker? Here it would be passed to MSVC's linker which I guess doesn't understand it.